### ISSUE TITLE
Syntax error in ResourceDetachController:43

### ISSUE THREAD

**Issue by sg-ip** [laravel/nova-issues#3464]
- PHP Version: 7.2
- Laravel Version: 7
- Nova Version: 3.24.0-3.27.0

### Description:
Detaching resources does not work because of a syntax error in`src/Http/Controllers/ResourceDetachController.php`
```php
                if (empty($inPivots) || in_array($pivot->getAttribute($accessorKeyName), $inPivots)) {
                    $this->deletePivot(
                        $request, $pivot, $model, $parent, // <-- the trailing comma in the parameter list is the problem
                    );
                }
```

Updating to 3.27 didn't help, I also deleted vendor/laravel/nova and cleared the composer cache, then ran `composer install`, making sure I don't have a stale version of that file.

**Comment by crynobone** [comment]
PHP 7.2 with Laravel 7?

**Comment by sg-ip** [comment]
Yes, I added that information.

**Comment by crynobone** [comment]
Fixed in v3.28.0: https://nova.laravel.com/releases/3.28.0
